packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx

```diff
@@ -137,91 +142,86 @@ export function SewerChart({
         source: text.source,
       }}
       description={text.description}
+      onSelectTimeframe={(timeframe) => setSewerTimeframe(timeframe)}
```

Comment:
Optional; but you could use the shorthand version:

```suggestion
      onSelectTimeframe={setSewerTimeframe}
```

Reply:
Changed.